deploy/00_deploy_protocol.js

```diff
@@ -0,0 +1,40 @@
+// deploy/00_deploy_my_contract.js
+const hre = require("hardhat");
+
+module.exports = async ({ getNamedAccounts, deployments }) => {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+  console.log("deployer:", deployer);
+  const metricToken = await deploy("MetricToken", {
+    from: deployer,
+    args: [deployer],
+    log: true,
+  });
+
+  const chef = await deploy("Chef", {
```

Comment:
will have to change to TopChef as Chef will be an abstract contract that others will inherit from